discord/role.py

```diff
@@ -66,41 +66,48 @@ class RoleTags:
         The bot's user ID that manages this role.
     integration_id: Optional[:class:`int`]
         The integration ID that manages the role.
-    subscription_listing_id: Optional[:class:`int`]
+    boost_listing_id: Optional[:class:`int`]
         The ID of this role's subscription SKU and listing.
-
+    subscription_listing_id: Optional[:class:`int`]
+        An alias for `boost_listing_id`
         .. versionadded:: 2.2
     """
 
     __slots__ = (
         'bot_id',
         'integration_id',
-        '_premium_subscriber',
+        '_nitro_subscriber',
         '_available_for_purchase',
+        'boost_listing_id',
         'subscription_listing_id',
         '_guild_connections',
     )
 
     def __init__(self, data: RoleTagPayload):
         self.bot_id: Optional[int] = _get_as_snowflake(data, 'bot_id')
         self.integration_id: Optional[int] = _get_as_snowflake(data, 'integration_id')
-        self.subscription_listing_id: Optional[int] = _get_as_snowflake(data, 'subscription_listing_id')
+        self.boost_listing_id: Optional[int] = _get_as_snowflake(data, 'subscription_listing_id')
+        self.subscription_listing_id: Optional[int] = self.boost_listing_id
 
         # NOTE: The API returns "null" for this if it's valid, which corresponds to None.
         # This is different from other fields where "null" means "not there".
         # So in this case, a value of None is the same as True.
         # Which means we would need a different sentinel.
-        self._premium_subscriber: bool = data.get('premium_subscriber', MISSING) is None
+        self._nitro_subscriber: bool = data.get('premium_subscriber', MISSING) is None
```

Comment:
Why was this changed?

Reply:
Oops, forgot to change that from a old PR

Reply:
It is difficult when editing files in mobile git lmao, going to change it asap